### ISSUE TITLE
InvalidReportError: Empty report - must contain at least one asset/block

### ISSUE THREAD

**Issue by giswqs** [datapane/datapane#179]
datapane v0.13.1

All the examples throw the same error.
https://docs.datapane.com/reports/blocks/plots-and-visualizations#folium

```python
import folium
import datapane as dp 

m = folium.Map(location=[45.5236, -122.6750])

dp.Report(dp.Plot(m)).upload(name='folium_map')
```
![image](https://user-images.githubusercontent.com/5016453/147392695-65100519-57fe-46e8-b01d-c6db979320a2.png)

**Comment by mands** [comment]
Hi there,

Thanks for raising the issue, we can confirm and reproduce the issue.

We aim to have a fix out in the next few days, in the meantime as a workaround if you add another block, even just a bit of text, to the report it should work, e.g.

```python
dp.Report(dp.Plot(plot), "my plot")
```

**Comment by mands** [comment]
thanks for raising the issue - this has been fixed in version 0.13.2
